clang/test/SemaSYCL/num_simd_work_items_device.cpp

```diff
@@ -70,7 +76,8 @@ int main() {
         []() [[intel::num_simd_work_items(-42)]]{}); // expected-error{{'num_simd_work_items' attribute requires a positive integral compile time constant expression}}
 
     h.single_task<class test_kernel6>(
-        []() [[intel::num_simd_work_items(1), intel::num_simd_work_items(2)]]{}); // expected-warning{{attribute 'num_simd_work_items' is already applied with different parameters}}
+        []() [[intel::num_simd_work_items(1), intel::num_simd_work_items(2)]]{}); // expected-warning{{attribute 'num_simd_work_items' is already applied with different parameters}} \
```

Comment:
one nit.
There is one extra \ at the end of the diagnostic message.

// expected-warning{{attribute 'num_simd_work_items' is already applied with different parameters}} \